### ISSUE TITLE
OctoPrint event / settings sync fails

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#187]
## **Describe the bug**
OctoPrint's NATs event publisher throws the following error:

```
Jan  3 03:11:49 printnanny octoprint[682]: ^[[33m2023-01-03 03:11:49,924 - octoprint.plugins.octoprint_nanny - WARNING - PrintNanny Cloud NATS connection is not initialized, skipping event: ClientAuthed^[[0m
Jan  3 03:11:50 printnanny octoprint[682]: ^[[31m2023-01-03 03:11:50,150 - octoprint.plugins.octoprint_nanny.utils - ERROR - Failed to get printnanny config cmd=['/usr/bin/printnanny', 'config', 'show', '--format', 'json'] returncode=2 stdout= stderr=error: Found argument 'config' which wasn't expected, or isn't valid in this context
```

This command was renamed to `printnanny settings show`.

## Issue.txt
```
-----------------------
Build Info:  |
-----------------------
IMAGE_NAME = printnanny-release-image-raspberrypi4-64-20221231215411
DISTRO_PRETTY = PrintNanny Linux 0.5.0 (Emerald)
VARIANT_NAME = PrintNanny OS Core
VARIANT_ID = core
DISTRO = printnanny
DISTRO_NAME = PrintNanny Linux
DISTRO_VERSION = 0.5.0
DISTRO_CODENAME = Emerald
ID_LIKE = BitsyLinux
HOME_URL = https://printnanny.ai
BUG_REPORT_URL = https://github.com/bitsy-ai/printnanny-os/issues
YOCTO_VERSION = 4.0.2
YOCTO_CODENAME = Langdale
MACHINE = raspberrypi4-64
TUNE_PKGARCH = cortexa72
-----------------------
Layer Revisions:      |
-----------------------
meta              = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-poky         = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-yocto-bsp    = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-raspberrypi  = libcamera-v0.0.3:104a8b9290ab4da92c8f693e2fdc42a6396975d2
meta-oe           = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-python       = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-multimedia   = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-bitsy        = main:cff99666a1f129e5944551a7643d3725bb5a1821
meta-neural-network = master:fed68f0a659c9b6431985c156141dc28dc4c2275
meta-printnanny   = main:cff99666a1f129e5944551a7643d3725bb5a1821
meta-networking   = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-filesystems  = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-initramfs    = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-webserver    = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-swupdate     = master:5e611f44db1b9333342ed61d77a3c3ed7be69c2b
workspace         = langdale:a3e3b740e140d036122f7b11e2ac452bda548444
meta-microcontroller = master:9f396e595b2d639be7f2b494f4a61e48d9495058
```

**Comment by leigh-johnson** [comment]
Released as plugin version 0.15.2: https://github.com/bitsy-ai/octoprint-nanny-plugin/releases/tag/0.15.2

**Comment by leigh-johnson** [comment]
Distro release: https://github.com/bitsy-ai/meta-bitsy/pull/343
